telco-examples/mgmt-cluster/aarch64/eib/mgmt-cluster-arm-singlenode.yaml

```
@@ -27,46 +27,46 @@ kubernetes:
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn-crd
-        version: 104.2.0+up1.7.1
-        repositoryName: rancher-charts
+        version: 105.1.0+up1.7.2
+        repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn
-        version: 104.2.0+up1.7.1
-        repositoryName: rancher-charts
+        version: 105.1.0+up1.7.2
+        repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
-      - name: metal3
+      - name: metal3-chart
```

```suggestion
      - name: metal3
```

The name in the upstream repo is just `metal3`.

---

yes you're right it's the same that it was before